From ee18f89c49927f6696df2d557837acc9bfd71f71 Mon Sep 17 00:00:00 2001
From: "Wladimir J. van der Laan" <laanwj@gmail.com>
Date: Fri, 3 Feb 2017 09:30:53 +0000
Subject: [PATCH 2/2] Revert "Subject: [PATCH] etnaviv: add RGBX/RGBA pixel
 formats"
Organization: O.S. Systems Software LTDA.

This reverts commit 89bb5c79e29613ad9a4e43d670654e98a220fc60.

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
Signed-off-by: Fabio Estevam <festevam@gmail.com>
---
 src/gallium/drivers/etnaviv/etnaviv_format.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/gallium/drivers/etnaviv/etnaviv_format.c b/src/gallium/drivers/etnaviv/etnaviv_format.c
index 0794603b2f..693558476b 100644
--- a/src/gallium/drivers/etnaviv/etnaviv_format.c
+++ b/src/gallium/drivers/etnaviv/etnaviv_format.c
@@ -141,8 +141,6 @@ static struct etna_format formats[PIPE_FORMAT_COUNT] = {
    V_(R16G16_SSCALED, SHORT,          NONE),
    V_(R16G16_FLOAT,   HALF_FLOAT,     NONE),
 
-   V_(A8B8G8R8_UNORM,   UNSIGNED_BYTE, NONE),
-
    V_(R8G8B8A8_UNORM,   UNSIGNED_BYTE, A8B8G8R8),
    V_(R8G8B8A8_SNORM,   BYTE,          A8B8G8R8),
    _T(R8G8B8X8_UNORM,   X8B8G8R8,      X8B8G8R8),
@@ -151,9 +149,6 @@ static struct etna_format formats[PIPE_FORMAT_COUNT] = {
    V_(R8G8B8A8_USCALED, UNSIGNED_BYTE, A8B8G8R8),
    V_(R8G8B8A8_SSCALED, BYTE,          A8B8G8R8),
 
-   _T(R8G8B8A8_UNORM, A8B8G8R8, A8B8G8R8),
-   _T(R8G8B8X8_UNORM, X8B8G8R8, X8B8G8R8),
-
    _T(B8G8R8A8_UNORM, A8R8G8B8, A8R8G8B8),
    _T(B8G8R8X8_UNORM, X8R8G8B8, X8R8G8B8),
 
-- 
2.12.0

